packages/cozy-harvest-lib/src/helpers/accounts.js

```diff
@@ -11,7 +13,8 @@ export const prepareAccountData = (konnector, data) => {
   // For now we are just ensuring legacy
   return {
     auth: data,
-    account_type: konnector.slug
+    account_type: konnector.slug,
+    identifier: Manifest.getIdentifier(konnector.fields)
```

Comment:
sanitized here for example, or higher up

Reply:
I'll think about it. 👍